shellscripts/createVaultedSshKeyPair.bash

```diff
@@ -74,7 +74,7 @@ writeEncrypted "$encrypted" "$varname" "$outputToFile"
 workDir=/tmp/_keys
 rm -rf $workDir
 mkdir $workDir
-ssh-keygen -t rsa -b 4096 -C "$email" -N "$password" -f $workDir/${varname_prefix}_id_rsa
+ssh-keygen -t ed25519 -C "$email" -N "$password" -f $workDir/${varname_prefix}_id_rsa
```

Comment:
suffix should not be `rsa` any more then - should be `ed` I assume?

or we decide to drop the suffix so that we can easily change key type without worrying about variable names - I'm happy to do that

Reply:
When I run `ssh-keygen -t ed25519` locally the suggested name is id_ed25519. Maybe we could also consider opting out of the suffix altogether?

Reply:
yes that is what I think is best - lets get rid of suffix altogether